src/aws/lambda-convert/src/convertSharedLiveMedia.spec.js

```diff
@@ -0,0 +1,65 @@
+const fs = require("fs");
+
+process.env.S3_DESTINATION_BUCKET = "destination_bucket";
+
+const testDirectory = `${__dirname}/../testfiles/`;
+const pdfFile = `${testDirectory}sample.pdf`;
+const svgFiles = [
+  `${testDirectory}sample-1.svg`,
+  `${testDirectory}sample-2.svg`,
+  `${testDirectory}sample-3.svg`,
+];
+
+// Don't pollute tests with logs intended for CloudWatch
+jest.spyOn(console, "log");
+
+// Mock the AWS SDK calls used in encodeTimedTextTrack
+const mockGetObject = jest.fn();
+const mockPutObject = jest.fn();
+jest.mock("aws-sdk", () => ({
+  S3: function () {
+    this.getObject = mockGetObject;
+    this.putObject = mockPutObject;
+  },
+}));
+
+const convertSharedLiveMedia = require("./convertSharedLiveMedia");
+
+describe("lambda-convert/src/convertSharedLiveMedia", () => {
+  beforeEach(() => {
+    console.log.mockReset();
+    jest.resetAllMocks();
+  });
+  it("converts uploaded sharedlivemedia", async () => {
+    mockGetObject.mockReturnValue({
+      promise: () =>
+        new Promise((resolve) => resolve({ Body: fs.readFileSync(pdfFile) })),
+    });
+    mockPutObject.mockReturnValue({
+      promise: () => new Promise((resolve) => resolve()),
+    });
+
+    const nbPages = await convertSharedLiveMedia(
+      "ed08da34-7447-4141-96ff-5740315d7b99/sharedlivemedia/c5cad053-111a-4e0e-8f78-fe43dec11512/1638403200.pdf",
+      "source bucket"
+    );
+
+    expect(nbPages).toBe(3);
+    expect(mockPutObject).toHaveBeenCalledTimes(nbPages);
+    expect(mockPutObject).toHaveBeenCalledWith({
+      Body: fs.readFileSync(svgFiles[0]),
+      Key: "ed08da34-7447-4141-96ff-5740315d7b99/sharedlivemedia/1638403200.pdf_1.svg",
```

Comment:
IMO the `.pdf` extension should be removed.

Reply:
Something like this would be ok ?
```
Key: "ed08da34-7447-4141-96ff-5740315d7b99/sharedlivemedia/1638403200_1.svg",
```

Reply:
Yes it is and I will have to change this url in the serializer :+1:

Reply:
Hmm wait there is other think wrong in your url.

It should be `ed08da34-7447-4141-96ff-5740315d7b99/sharedlivemedia/c5cad053-111a-4e0e-8f78-fe43dec11512/1638403200.pdf_1.svg`

it's `{video.id}/sharedlivemedia/{shared_live_media.id}/{stamp}_{page}.svg`

Also,